RecoParticleFlow/PFProducer/python/particleFlowSimParticle_cfi.py

```diff
@@ -35,3 +29,8 @@
     #retrieving fastSim SimHits                                     
     fastSimProducer = cms.untracked.InputTag('fastSimProducer','EcalHitsEB')
 )
+
+from FastSimulation.Event.ParticleFilter_cfi import ParticleFilterBlock
```

Comment:
this conflicts with #26016

Reply:
I see. I will just wait until 26016 then?

Reply:
OK.
@Dr15Jones 
did you have a timeline in mind for resolving #26016 
?

Reply:
#26061 is awaiting reconstruction review. Once that pull request is merged, I was going to move FastSimulation/BaseParticlePropagator to CommonTools/BaseParticlePropagator. That solves almost all the dependencies. 

It does NOT break the dependency of FastSimulation on PFSimParticleProducer since that is using more of the 'heart' of fast simulation (it is also a 'Sim' related module).

Reply:
@Dr15Jones 
Did I read your last message correctly that a new dependence in this PR of ```RecoParticleFlow/PFProducer/python/particleFlowSimParticle_cfi ``` on ```FastSimulation/Event``` is OK and will not actually conflict with activities related/expected from the resolution of #26016 "Break Reconstructions dependency on FastSimulation"?

Reply:
I was only looking at build time dependencies, not python ones. Therefore this should have no effect on that restructuring. Whether or not adding such a dependency here is a good idea is another question.

Reply:
Sounds good. We can then proceed with the review of this PR.
@hatakeyamak are you done with your edits in this PR?

Reply:
Yes.